src/app/donation-start/donation-start-form/donation-start-form-parent.component.ts

```diff
@@ -894,6 +896,7 @@ export class DonationStartFormParentComponent implements AfterContentChecked, Af
       this.stepper.next();
       return;
     }
+    this.stripePaymentMethodReady = this.paymentMethodChecked || this.stripeManualCardInputValid;
```

Comment:
Why should clicking next affect whether we consider the payment method to be ready?